docs/sphinx/source/whatsnew/v0.6.0.rst

```diff
@@ -11,9 +11,37 @@ API Changes
 
 Enhancements
 ~~~~~~~~~~~~
-* Add sea surface albedo in irradiance.py (:issue:`458`)
-* Implement first_solar_spectral_loss in modelchain.py (:issue:`359`)
-* Clarify arguments Egref and dEgdT for calcparams_desoto (:issue:`462`)
+* Add sea surface albedo in ``irradiance.py`` (:issue:`458`)
+* Implement :meth:`~pvlib.modelchain.ModelChain.first_solar_spectral_loss`
+  in ``modelchain.py`` (:issue:`359`)
+* Clarify arguments ``Egref`` and ``dEgdT`` for
+  :func:`~pvlib.pvsystem.calcparams_desoto` (:issue:`462`)
+* Implement a reliable "gold" implementation of the single diode model (SDM)
```

Comment:
I still find this whatsnew text to be confusing.  I'd put the following in 2nd position in the list

* Implement new methods `brentq` and `newton` for solving for points on the single diode equation. `brentq` uses a bisection method (Brent, 1973) that may be slow but guarantees a solution. `newton` implements the Newton-Raphson method that is faster but is not guaranteed to converge, however, `newton` should be safe for well-behaved IV curves. (:issues:`408`)

Reply:
✔️